docs/tutorials.rst

```diff
@@ -103,6 +103,156 @@ Which should print:
     hello, world
 
 
+Install packages slices into a ROCK
+===================================
+
+In this tutorial you will create a lean ROCK that contains a fully functional OpenSSL installation, and you will verify
+that it is functional by loading the ROCK into Docker and using it to validate the certificates of the Ubuntu website.
+
+Prerequisites
+-------------
+
+- snap enabled system (https://snapcraft.io)
+- LXD installed
+- skopeo installed (https://github.com/containers/skopeo)
+- Docker installed (https://docs.docker.com/get-docker/)
+- a text editor
+
+
+Install Rockcraft
+-----------------
+
+Install Rockcraft on your host:
+
+.. code-block:: sh
+
+    snap install rockcraft --classic --edge
+
+
+Project Setup
+-------------
+
+Create a new directory, write the following into a text editor and save it as ``rockcraft.yaml``:
+
+.. code-block:: yaml
+
+    name: chisel-openssl
+    summary: OpenSSL from Chisel slices
+    description: A "bare" ROCK containing an OpenSSL installation created from Chisel slices.
+    license: Apache-2.0
+
+    version: "0.0.1"
+    base: bare
+    build_base: "ubuntu:22.04"
+    entrypoint: [/usr/bin/openssl]
+    platforms:
+      amd64:
+
+    env:
+      - SSL_CERT_FILE: /etc/ssl/certs/ca-certificates.crt
+
+    parts:
+      openssl:
+        plugin: nil
+        stage-packages:
+          - openssl_bins
+          - ca-certificates_data
+
+Note that this Rockcraft file uses the ``openssl_bins`` and ``ca-certificates_data`` Chisel slices to generate an image
```

Comment:
I tried to strike an "explanation" balance here: I wanted to highlight the fact that this uses chisel slices, and motivate it a little bit, but in Diátaxis Tutorials shouldn't be explanation-heavy (ideally they should be explanation-less). This is what I came up with